mls-rs-uniffi/tests/custom_storage_sync.py

```diff
@@ -0,0 +1,84 @@
+from dataclasses import dataclass, field
+
+from mls_rs_uniffi import CipherSuite, generate_signature_keypair, Client, \
+    GroupStateStorage, GroupState, EpochRecord, ClientConfig, ProtocolVersion
+
+@dataclass
+class GroupStateData:
+    state: bytes
+    epoch_data: list[EpochRecord] = field(default_factory=list)
+
+class PythonGroupStateStorage(GroupStateStorage):
+    def __init__(self):
+        self.groups: dict[str, GroupStateData] = {}
+
+    def state(self, group_id: bytes):
+        group = self.groups.get(group_id.hex())
+
+        if group == None:
+            return None 
+
+        return group.state
+
+    def epoch(self, group_id: bytes, epoch_id: int):
+        group = self.groups[group_id.hex()]
+
+        if group == None:
+            return None
```

Comment:
I think it should be `self.groups.get(...`? The current code throws an error if the group is not found?

Reply:
Thanks! The code was merely moved, but now this is fixed as well :smile:

Reply:
Well, good thing this made us notice. Thanks!